src/renderer/components/dialog-add-version.tsx

```diff
@@ -168,25 +190,33 @@ export class AddVersionDialog extends React.Component<
   }
 
   private renderPath(): JSX.Element | null {
-    const { folderPath, isValidElectron } = this.state;
+    const { isValidElectron, folderPath, existingLocalVersion } = this.state;
+    const shouldSetToPreviousVersion = isValidElectron && existingLocalVersion;
```

Comment:
```suggestion
    const canSwitch = isValidElectron && existingLocalVersion;
```

For consistency with other suggested change